Bottom: fa05dccbd564b9f45a501d8081218ba68af25a87
Top:    07bc5862ed6c2545c842cffec9d51bf25b79fb40
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-08-20 10:19:47 -0400

Refresh of sgtl5000

---

diff --git a/arch/arm/boot/dts/sun7i-a20-cubietruck.dts b/arch/arm/boot/dts/sun7i-a20-cubietruck.dts
index 2267a54..3cb91f2 100644
--- a/arch/arm/boot/dts/sun7i-a20-cubietruck.dts
+++ b/arch/arm/boot/dts/sun7i-a20-cubietruck.dts
@@ -206,15 +206,13 @@
 	sound {
 		compatible = "simple-audio-card";
 		simple-audio-card,format = "i2s";
-		simple-audio-card,bitclock-master = <&dailink0_master>;
-		simple-audio-card,frame-master = <&dailink0_master>;
 
 		simple-audio-card,cpu {
 			sound-dai = <&iis0>;
 		};
 
-		dailink0_master: simple-audio-card,codec {
-			system-clock-frequency = <24000000>;
+		simple-audio-card,codec {
+			clock = <&iis0>;
 			sound-dai = <&sgtl5000>;
 		};
 	};
